### PR TITLE
added composio in list.md file

### PR DESCRIPTION
**Pull request by githubashutoshsoni**
Thanks for contributing to the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com/) :smiley: :tada:! Before submitting your pull request, please check off as many of the items below as you can:

1. [x] I have read the [Contributing.md](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/docs/contributing.md) file and formatted this PR correctly
2. [x] I'm not adding a company from the blocklist
3. [x] I make sure to fix things promptly if an error or suggestion comes up

Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

### CODE REVIEW

File: docs/list.md [hunk 1]
```diff
@@ -67,6 +67,14 @@ No one currently! Interested in having a prominent placement for your open sourc
 - **How to sign up**: No special signup known.
 - **Notes**: [Visit Cloudinary](https://cloudinary.com/blog/hacktoberfest-open-source-celebration-cloudinary) for the list of eligible libraries and SDKs.
 
+#### **composio**
+
+- **Swag**: Stickers, T-shirt
+- **Requirements**: Make 4 good PRs against 'hacktoberfest' topics in any of the Composio repositories.
+- **How to sign up**: Register for Hacktoberfest [here](https://hacktoberfest.com/).
```
Comment (crweiner) [inline docs/list.md:74]: This line is not to register for Hacktoberfest, rather how to register to receive this particular company's swag. Is there a signup for Composio themselves?

File: docs/list.md [hunk 2]
```diff
@@ -67,6 +67,14 @@ No one currently! Interested in having a prominent placement for your open sourc
...
+- **Issues**: Optional link to Hacktoberfest tagged issues in Composio repositories.
```
Comment (crweiner) [inline docs/list.md:75]: It looks like this is incomplete. Can you please link to the `hacktoberfest` tagged issues?

File: docs/list.md [hunk 3]
```diff
@@ -67,6 +67,14 @@ No one currently! Interested in having a prominent placement for your open sourc
 - **How to sign up**: No special signup known.
 - **Notes**: [Visit Cloudinary](https://cloudinary.com/blog/hacktoberfest-open-source-celebration-cloudinary) for the list of eligible libraries and SDKs.
 
+#### **composio**
```
Comment (crweiner) [inline docs/list.md:70]: Should the title be capitalized?

**Review by crweiner — COMMENTED** [review]
Hi @githubashutoshsoni, thanks for your PR! It looks like this is a little incomplete, could you please review what I added and fix them?

Let me know if you have any questions, thanks!

File: docs/list.md (hunk 3 above)
Comment (githubashutoshsoni) [reply docs/list.md:70]: Yes, seems right let me change this

File: docs/list.md (hunk 1 above)
Comment (githubashutoshsoni) [reply docs/list.md:74]: No they don't have it's a generic link. I guess I can update this to https://hacktoberfest.com/participation/ as how they have it in there HACTOBERFEST_MD file?

File: docs/list.md (hunk 2 above)
Comment (githubashutoshsoni) [reply docs/list.md:75]: sure

**Comment by githubashutoshsoni** [comment]
Hi @crweiner, Can you please check again? seems like there was a conflict I resolved it anything specific you want me to check please let me know.

**Review by githubashutoshsoni — COMMENTED** [review]
completed the changes as per request

**Comment by crweiner** [comment]
Hi @githubashutoshsoni, thanks for your PR! Please be sure that you are making sure that you double check your submission for some basic things, like adding your entry in alphabetical order. I've made some changes and am happy to get this live.

Thank you again @vimode for suggesting some edits as well!

**Pull request merged**